interface/main/mobile/SMS.php

```diff
@@ -0,0 +1,356 @@
+                           src="#"
+                           id="sms_search"
+                           onclick="search4SMS()"
+                           value="<?php echo xla('Search'); ?>" />
+                </div>
+                <div id="NEW_SMS" class="line_bottom_style">
+                    <input type="submit" class="fa fa-arrow-up btn btn-primary"
+                           style="position:relative;float:right;top: -2px;margin-right:15px;
+                                                    padding: 6px;font-weight:700;color:white;font-size: 1em;"
+                           src="#"
+                           id="sms_submit"
+                           onclick="sendSMS();"
+                           value="<?php echo xla('Send'); ?>" />
+                    <input type="hidden" id="local_pid" value="" />
+                    <input type="hidden" id="msg_last_updated" value="0" />
+                    <input type="hidden" id="medex_uid" value="" />
+                    <textarea style="width: calc(100% - 80px);
+                                                        height:29px;
+                                                        margin: 2px 5px;
+                                                        background-color: #fff;
+                                                        border-radius: 5px;
+                                                        padding: 4px 6px 3px;
+                                                        position: relative;
+                                                        float: right;
+                                                        resize: none;" value=""
+                                  name="sms_text" id="sms_text" placeholder="<?php echo xla('Text Message'); ?>" autofocus></textarea>
+                </div>
+            </div>
+        </div>
+    
+</div>
+<script>
+    var pid;
+    function goForward(pid) {
+        top.restoreSession();
+        $("#NEW_SMS").show();
+        $('#search_data_right').hide();
+        $('#message_data_right').show();
+        $("#new_SMS_icon").show();
+    
+        if (pid =='') return false;
+        $('#message_data_right').html('<div class="text-center">\n'+
+'                        <i class="fa fa-spinner fa-pulse fa-fw" style="font-size: 140px; color: #0000cc; padding: 20px"></i>\n'+
+'                        <h2><?php echo xlt('Loading data'); ?>...</h2>\n'+
+'                    </div>');
+        
+        
+        $("#message_data div").css("background-color", "transparent");
+        //get the name/number for display
+        $.ajax({
+                   type: "POST",
+                   url: "m_save.php?go=sms_search&term="+pid
+               }).done(function(result) {
+                console.log(result);
+                results = JSON.parse(result);
+                if (!results.mobile) {
+                    results.mobile = results.home_phone + "  - <?php echo xlt('Home'); ?>";
```

Comment:
```
results.mobile = results.home_phone + "  - " + <?php echo xlj('Home'); ?>;
```
(this is a function that will escape correctly in pure js parts of scripts; note part of the escaping is that it adds quotes to the end, which is why do not need them above)